### PR TITLE
refactor(deps): Use trivy pkgs instead of defsec

### PR DESCRIPTION
**Pull request by simar7**

### CODE REVIEW

**Comment by simar7** [comment]
Update when https://github.com/aquasecurity/trivy/pull/6109/ is merged

**Comment by simar7** [comment]
@nikpivkin could you take a look at reviewing this?

File: checks/cloud/aws/accessanalyzer/enable_access_analyzer.go [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-	"github.com/aquasecurity/defsec/pkg/severity"
-	"github.com/aquasecurity/defsec/pkg/state"
-	defsecTypes "github.com/aquasecurity/defsec/pkg/types"
 	"github.com/aquasecurity/trivy-policies/pkg/rules"
+	"github.com/aquasecurity/trivy/pkg/iac/framework"
+	"github.com/aquasecurity/trivy/pkg/iac/providers"
+	"github.com/aquasecurity/trivy/pkg/iac/scan"
+	"github.com/aquasecurity/trivy/pkg/iac/severity"
+	"github.com/aquasecurity/trivy/pkg/iac/state"
+	defsecTypes "github.com/aquasecurity/trivy/pkg/iac/types"
```
Comment (nikpivkin) [inline checks/cloud/aws/accessanalyzer/enable_access_analyzer.go:10]: I think `defsecTypes` could be renamed `trivyTypes`.

**Comment by simar7** [comment]
Sorry @nikpivkin I superseded this PR with https://github.com/aquasecurity/trivy-policies/pull/80 - that one also includes the use of trivy packages as it was required to do that with the cmd/ binaries.

File: checks/cloud/aws/accessanalyzer/enable_access_analyzer.go (hunk 1 above)
Comment (simar7) [reply checks/cloud/aws/accessanalyzer/enable_access_analyzer.go:10]: I will apply this suggestion in this PR https://github.com/aquasecurity/trivy-policies/pull/80 thanks.

**Pull request closed without merging**